### DISCUSSION TITLE
Autocomplete removes my previous input on focus

### DISCUSSION THREAD

**Discussion by eugenegorelikov** [TarekRaafat/autoComplete.js#186] [Q&A] [answered]
In your demo  https://tarekraafat.github.io/autoComplete.js/demo/ autocomplete retains the inputs, however in my implementation for some reason whenever i place the cursor into a text input with another (previous) input, the text input gets emptied. Could anyone please advise me a setting to allow the previous inputs to remain? Thank you!

**Comment by TarekRaafat** [comment, marked as answer]
Hello @eugenegorelikov,

My bad, I did not get your question clearly.

Could you please share more details on your issue?

**Reply by eugenegorelikov** [reply]
Apologies, please disregard the question. I was dealing with a form that someone created before and there was an onclick event assigned to it. So if anyone approaches you with the same question, please point them out that they might have other scripts doing things.
